homeassistant/helpers/entity_registry.py

```diff
@@ -34,24 +37,36 @@
 class RegistryEntry:
     """Entity Registry Entry."""
 
+    # pylint: disable=no-member
+
     entity_id = attr.ib(type=str)
     unique_id = attr.ib(type=str)
     platform = attr.ib(type=str)
     name = attr.ib(type=str, default=None)
     disabled_by = attr.ib(
         type=str, default=None,
         validator=attr.validators.in_((DISABLED_HASS, DISABLED_USER, None)))
-    domain = attr.ib(type=str, default=None, init=False, repr=False)
+    update_listeners = attr.ib(type=list, default=attr.Factory(list),
+                               repr=False)
+    domain = attr.ib(type=str, init=False, repr=False)
 
-    def __attrs_post_init__(self):
-        """Computed properties."""
-        object.__setattr__(self, "domain", split_entity_id(self.entity_id)[0])
+    @domain.default
```

Comment:
What does this mean?

Reply:
That is for attr library and return the default value of this member

Reply:
Thanks!

Reply:
I felt that this was cleaner than the post init part. When default is called, you have access to all attributes that are defined before domain